### PR TITLE
Add `empty` to `run-testenv`

### PR DESCRIPTION
**Pull request by TimeEngineer**

### CODE REVIEW

File: cli/src/run_testenv.rs [hunk 1]
```diff
@@ -39,6 +39,9 @@ pub struct RunTestenv {
     /// When this process stops, the server will be automatically killed
     #[arg(long)]
     main_process_id: u32,
+    /// Skip initialization
+    #[arg(short, long)]
+    empty: Option<bool>,
```
Comment (FirelightFlagboy) [inline cli/src/run_testenv.rs:44]: You can configure `clap` to use a default value
 
```suggestion
    /// Skip initialization
    #[arg(short, long, default_value_t)]
    empty: bool,
```

The code above will set `empty` to `false` when the value wasn't provided in the CLI

https://docs.rs/clap/latest/clap/_derive/index.html#arg-attributes

Comment (TimeEngineer) [reply cli/src/run_testenv.rs:44]: Nice

**Review by vxgmichel — CHANGES_REQUESTED** [review]
I would not expect a server to be started when the `--empty` option is passed, but here's what I got:
```python
± source ./cli/run_testenv.sh --empty
Running server with the process id: 67969 on port: 65247
```
IMO an empty env is more similar to a new user installing parsec on their machine getting invited with a link. That means that the corresponding server is already running another environment (typically another shell running `run_testenv.sh` without `--empty`)

Also I wouldn't make `empty` an argument to `run_local_organization` as we are not running an empty organization but we're creating an empty env.

**Review by vxgmichel — APPROVED** [review]
LGTM :+1:

**Pull request merged**